api/DOMTokenList.json

```diff
@@ -610,10 +610,10 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": true
+                "version_added": "6.2"
```

Comment:
The WebKit editions of Safari 6.1 and 6.2 are so similar, we just set it to 6.1.  Can we bump this down to 6.1?

Reply:
Done.